src/main/java/roomescape/member/domain/Email.java

```diff
@@ -0,0 +1,43 @@
+package roomescape.member.domain;
+
+import jakarta.persistence.Embeddable;
+import java.util.regex.Pattern;
+
+@Embeddable
+public class Email {
+    private static final int MAX_EMAIL_LENGTH = 20;
+    private static final Pattern EMAIL_PATTERN = Pattern.compile(
+            "^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$"
+    );
+
+    private String value;
+
+    protected Email() {
+
+    }
+
+    public Email(final String value) {
+        validateEmail(value);
+        this.value = value;
+    }
+
+    private void validateEmail(final String value) {
+        if (value == null || value.length() == 0) {
```

Comment:
아래와 같이 변경할 수 도 있습니다. 

```suggestion
        if (value == null || value.isEmpty() == 0) {
```

Reply:
[반영 완료] 

넵 수정하였습니다 ! 👍